magma/bits.py

```diff
@@ -157,10 +190,27 @@ def declare_binary_op(cls, op):
     @lru_cache(maxsize=None)
     def declare_compare_op(cls, op):
         N = len(cls)
+        python_op_name = {
```

Comment:
looks like there are a bunch of different mappings for these op names. Can we factor them out to somewhere common?